.pre-commit-config.yaml

```diff
@@ -10,6 +10,7 @@
     -   id: double-quote-string-fixer
     -   id: end-of-file-fixer
     -   id: flake8
+        exclude: migrations
```

Comment:
you'd better use full relative path `blog/migrations` to prevent accidentally ignoring smth important.